### PR TITLE
Profile information

### LINKED ISSUE #30: Establish default user profile image
Find image to use as default profile photo

### LINKED ISSUE #31: Research image upload ideas
Should we be using an image URL, or is there a simple way to let the user upload an image from their computer

### LINKED ISSUE #32: Display image in user profile
User should be able to see the default image in their profile

### LINKED ISSUE #35: Display user description to profile

### LINKED ISSUE #37: Create new path for Profile Editor Component
/profile/edit ??

### LINKED ISSUE #38: Build render for ProfileEditor
inputs should have default value of original user information to start

Component will need to hold state, and access/alter the global store

### LINKED ISSUE #39: Functionality for ProfileEditor
- [ ] Component should use an action to update the user's information on the server when the save changes button is pressed
- [ ] Back button should send user back to their profile

### PR DESCRIPTION
**Pull request by Kristiannmiller**
## Pull request type
Please check the type of change your PR introduces:
- [ ] Bugfix
- [X] Feature
- [X] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [X] Other (please describe): Styling, utilizing styled components

## Problem/feature
- closes #30 
- closes #31 
- closes #32 
- closes #35 
- closes #37 
- closes #38 
- closes #39 

## Solution
Added styled components used in other features to create a consistent style throughout the newly added features.

Added base component styles for Crates and Closet to plug in functionality later.

## Other changes (e.g. bug fixes, UI tweaks, small refactors)
Removed console.logs, added new components

## Checklist
- [X] Code compiles correctly
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] All tests passing
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: code/web/src/setup/routes/user.js [hunk 1]
```diff
@@ -35,5 +37,17 @@ export default {
     path: '/user/edit',
     component: Edit,
     auth: true
+  },
+
+  closet: {
```
Comment (taylorjohnson141) [inline code/web/src/setup/routes/user.js:42]: Thank You for adding this!

File: code/web/src/modules/user/api/actions.js [hunk 2]
```diff
@@ -79,10 +77,10 @@ export function editUser(userDetails){
     return axios.post(routeApi, mutation({
       operation: 'userUpdate',
       variables:{
-      id:userDetails.id, 
-      email:userDetails.email, 
-      picture:userDetails.picture, 
-      shippingAddress:userDetails.shippingAddress, 
+      id:userDetails.id,
```
Comment (taylorjohnson141) [inline code/web/src/modules/user/api/actions.js:80]: Thank you for getting rid of the console.logs

File: code/web/src/modules/user/Edit.js [hunk 3]
```diff
@@ -2,53 +2,137 @@ import React from 'react'
...
+                placeholder="Email"
+                required="required"
+                name="email"
+                autoComplete="off"
+                value={this.state.email}
+                onChange={this.onChange}
+              />
+
+              {/* Description */}
+              <Textarea
```
Comment (taylorjohnson141) [inline code/web/src/modules/user/Edit.js:98]: Nice job with the textarea!

**Review by taylorjohnson141 — APPROVED** [review]
Thank you for adding this. It looks really good!

**Review by Chriscastanuela — APPROVED** [review]

**Pull request merged**